resources/testdriver-actions.js

```diff
@@ -431,8 +431,9 @@
     this.actions = new Map();
   }
 
-  function checkPointerProperties(action, width, height, pressure, tangentialPressure,
-                                  tiltX, tiltY, twist, altitudeAngle, azimuthAngle) {
+  function setPointerProperties(actionData, width, height, pressure, tangentialPressure,
```

Comment:
It would make more sense to me to just name the argument `action` rather than immediately rebinding it.

Reply:
Done, thank you.